frontend/tests/e2e_tests/run

```diff
@@ -95,13 +81,13 @@ run_tests() {
         token=$(docker exec $containerid /usr/bin/tenantadm get-tenant --id $tenant | jq -r .tenant_token)
         jq --arg tenantToken "$token" '. + { "TenantToken": $tenantToken }' ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender.json > ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender-test.json
         # start a new client, using the obtained tenant token - starting it later (in the test suite) would require docker access in the test runner
-        docker run --name connect-client -d --network=gui-tests_mender \
+        docker run --name connect-client -d --network=gui-tests_default \
             -v ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender-test.json:/etc/mender/mender.conf \
             -v ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/mender-connect.json:/etc/mender/mender-connect.conf \
             mendersoftware/mender-client-docker-addons:mender-3.6.x
```

Comment:
I don't know about much about this script, but isn't this redefining what's already in the compose file?
```suggestion
       $COMPOSE_CMD run --name connect-client --detach client
```

Reply:
without the assigned network the client won't reach the backend as it gets a different network assigned + depending on the test setup the client is torn down based independently of the composition (for staging e.g.)

Reply:
The container is assigned to the `default` network by default where the backend now also lives.
Since run just starts a new container you can similarly kill it with `stop`, `rm` or even `kill` (if you're feeling brutal) compose commands.

Reply:
The compose network will be project scoped, so you have to assign it for the client to reach the backend.

My main goal is to have the client interactions aligned across the different environments the client is running in - staging won't have a compose attached.

Reply:
Ok, if you believe it is complicating things, then please disregard.